com.unity.ml-agents/Runtime/Communicator/RpcCommunicator.cs

```diff
@@ -43,6 +47,7 @@ internal class RpcCommunicator : ICommunicator
         Dictionary<string, ActionSpec> m_UnsentBrainKeys = new Dictionary<string, ActionSpec>();
 
 
+#if MLA_SUPPORTED_TRAINING_PLATFORM
```

Comment:
I think this was the merge problem I was worried about. I think you can either remove all the `#if MLA_SUPPORTED_TRAINING_PLATFORM` except the top-level one, or just keep master the master version.

Reply:
should be fixed now